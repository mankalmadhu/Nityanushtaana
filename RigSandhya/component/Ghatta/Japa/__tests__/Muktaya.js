import React from 'react';
import ಮುಕ್ತಾಯ from '../Muktaya';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya', () => 'ವಿಷಯ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಮುಕ್ತಾಯ", () => {
  expect(Renderer.create(
    <ಮುಕ್ತಾಯ />
  ).toJSON()).toMatchSnapshot();

});