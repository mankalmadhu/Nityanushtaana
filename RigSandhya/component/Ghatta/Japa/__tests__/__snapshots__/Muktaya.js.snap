// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಮುಕ್ತಾಯ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಮುಕ್ತಾಯ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಮುಕ್ತಾವಿದ್ರುಮ ಹೇಮ ನೀಲ ಧವಳಚ್ಛಾಯೈಃ ಮುಖೈಸ್ತ್ರೈಕ್ಷಣೈಃ ಯುಕ್ತಾಮಿ೦ದುನಿಬದ್ಧ ರತ್ನ ಮಕುಟಾ೦ ತತ್ವಾರ್ಥ ವರ್ಣಾತ್ಮಿಕಾ೦ । ಗಾಯತ್ರೀ೦ ವರದಾಭಯಾ೦ ಅ೦ಕುಶಕಶಾಃ ಶುಭ್ರ೦ ಕಪಾಲ೦ ಗದಾ೦ ಶ೦ಖ೦ ಚಕ್ರಮಥಾರವಿ೦ದ ಯುಗಳ೦ ಹಸ್ತೈರ್ವಹ೦ತೀ೦ ಭಜೇ ।
  </ವಿಷಯ>
</View>
`;
